cdap-cli/src/main/java/co/cask/cdap/cli/util/table/AltStyleTableRenderer.java

```diff
@@ -15,9 +15,11 @@
  */
```

Comment:
Seems like this shouldn't even be visible in this PR since the same commit is already merged to develop: https://github.com/caskdata/cdap/commit/12975d5daf310549c507d67eeba3769b638dad2d

Maybe you need to synchronize your fork with caskdata/cdap?